examples/webgl_occlusionqueries.html

```diff
@@ -0,0 +1,229 @@
+			var cameraControls;
+			var ambientLight, light;
+
+			var queryResult;
+
+			function init() {
+
+				var container = document.createElement( 'div' );
+				document.body.appendChild( container );
+
+				queryResult = document.createElement( 'div' );
+				document.body.appendChild( queryResult );
+				queryResult.style.position = 'absolute';
+				queryResult.style.margin = '0 auto';
+				queryResult.style.top = '100px';
+				queryResult.style.fontSize = '24px';
+				queryResult.style.left = '0';
+				queryResult.style.right = '0';
+
+				shadowQueryResult = document.createElement( 'div' );
+				document.body.appendChild( shadowQueryResult );
+				shadowQueryResult.style.position = 'absolute';
+				shadowQueryResult.style.margin = '0 auto';
+				shadowQueryResult.style.top = '130px';
+				shadowQueryResult.style.fontSize = '24px';
+				shadowQueryResult.style.left = '0';
+				shadowQueryResult.style.right = '0';
+
+				var canvasWidth = window.innerWidth;
+				var canvasHeight = window.innerHeight;
+
+				// CAMERA
+				camera = new THREE.PerspectiveCamera( 45, window.innerWidth / window.innerHeight, 0.1, 400 );
+				camera.position.set( 0, 8, 20 );
+
+				// LIGHTS
+				ambientLight = new THREE.AmbientLight( 0x333333 );
+
+				light = new THREE.DirectionalLight( 0xFFFFFF, 1.0 );
+				light.position.x = 2;
+				light.position.y = 7;
+				light.position.z = 10;
+				light.castShadow = true;
+				light.shadow.bias = -0.001;
+				light.shadow.mapSize.width = 1024;
+				light.shadow.mapSize.height = 1024;
+				light.shadow.camera.top = 10;
+				light.shadow.camera.bottom = -10;
+				light.shadow.camera.left = 10;
+				light.shadow.camera.right = -10;
+
+				// RENDERER
+				var canvas = document.createElement( 'canvas' );
+				var context = canvas.getContext( 'webgl2' );
+				renderer = new THREE.WebGLRenderer( { canvas: canvas, context: context } );
+				renderer.shadowMap.enabled = true;
+				renderer.setPixelRatio( window.devicePixelRatio );
+				renderer.setSize( canvasWidth, canvasHeight );
+				container.appendChild( renderer.domElement );
+
+				// EVENTS
+				window.addEventListener( 'resize', onWindowResize, false );
+
+				// CONTROLS
+				cameraControls = new THREE.OrbitControls( camera, renderer.domElement );
+
+				scene = new THREE.Scene();
+				scene.background = new THREE.Color( 0xAAAAAA );
+
+				scene.add( ambientLight );
+				scene.add( light );
+
+			}
+
+
+			function onWindowResize() {
+
+				var canvasWidth = window.innerWidth;
+				var canvasHeight = window.innerHeight;
+
+				renderer.setSize( canvasWidth, canvasHeight );
+
+				camera.aspect = canvasWidth / canvasHeight;
+				camera.updateProjectionMatrix();
+
+				render();
+
+			}
+
+
+			function render() {
+
+
+				renderer.render( scene, camera );
+
+			}
+
+			var sphere;
+			function createScene() {
+
+
+				var ground = new THREE.Mesh(new THREE.PlaneBufferGeometry(100, 100, 10, 10), new THREE.MeshStandardMaterial({ color: 0x22CC66, side: THREE.DoubleSide }));
```

Comment:
Please ensure correct code style. You should always have a space between parens.

Reply:
Done